qcelemental/physical_constants/__init__.py

```diff
@@ -3,12 +3,10 @@
 """
 
 # Ensure pint exists
-from importlib.util import find_spec
-spec = find_spec('pint')
-if spec is None:
+from ..importing import which_import
+if not which_import('pint', return_bool=True):
```

Comment:
Would rather leave these in `util` and not bother with the import here.

Reply:
then not going to get the soft "pint not found. do `pip install pint`" msg?

Reply:
Simply revert to the previous way of doing things?

Reply:
revised